docs/diagrams/sep6/deposit.md

```diff
@@ -0,0 +1,54 @@
+### Deposit
+
+This diagram illustrates how Anchors can provide deposit instructions to a user asynchronously. The flow starts from after the user has authenticated with the anchor via SEP-10 and provided some basic information about themselves. The example here requires the user deposits fund to the Anchor's bank account, but the flow is similar for other deposit methods. The `deposit-exchange` flow works similarly, but the Platform will additionally verify the quote requested or make a call to the Fee integration to update amounts.
+
+For more information on the deposit flow, see the [SEP-6](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md) specification.
+
+```mermaid
+sequenceDiagram
+  participant Bank
+  participant User
+  participant Wallet
+  participant Platform
+  participant Stellar
+  participant Anchor
+
+  Wallet->>Platform: GET /deposit
+  Platform-->>Platform: Creates deposit transaction with id abcd-1234 with status incomplete
+  Platform-->Wallet: Returns deposit response with id abcd-1234 and deposit instructions omitted
+  Platform--)Anchor: Sends an AnchorEvent with type transaction_created and transaction id abcd-1234
+  loop until timeout or status is pending_user_transfer_start
+    Wallet->>Platform: GET /transaction abcd-1234
+    Anchor--)Anchor: Evaluates whether additional KYC is required whenever KYC is updated (customer_updated events or PUT /customer callback)
```

Comment:
This wording is a bit confusing because KYC hasn't been updated at this point -- no KYC info has been provided (unless the wallet sent a `PUT /customer` request prior to `GET /deposit`, which isn't shown.

Reply:
I mentioned that SEP-10 and SEP-12 are omitted in the beginning paragraph.
> The flow starts from after the user has authenticated with the anchor via SEP-10 and provided some basic information about themselves

Do you have any suggestions on how to make it less confusing?

Reply:
Maybe we can just remove the "whenever KYC is updated ..." text, so its clear what the anchor is doing but we don't confuse the reader by mentioning an event that isn't shown prior (customer_updated events or customer callbacks).